shared/js/content-scripts/click-to-load.js

```diff
@@ -711,7 +711,7 @@
 
     function runLogin (entity) {
         enableSocialTracker(entity, true)
-        window.dispatchEvent(new CustomEvent(`Run${entity}Login`))
+        window.dispatchEvent(new CustomEvent(`ddg-ctp-${entity}-run-login`))
```

Comment:
Maybe lowercase and dasherize?

Reply:
Moved discussion here: https://app.asana.com/0/312629933896096/1201481772157517/f